verify_mapping_quality.pl

```diff
@@ -51,12 +51,12 @@ sub run {
     #set default number of cores
 	$cores = 1 if (not defined $cores);
 	#set default log directory to local directory    
-	$log_dir="./" if (not defined $log_dir);
+	$log_dir="./" if ((not defined $log_dir)||($log_dir eq '.')||(not -d $log_dir));
 	#set default minimum percent mapping
 	$min_map=$MIN_MAP if (not defined $min_map);
 	
 	#create the log file to print warnings to
-	open(my $log, '>', $log_dir.'mapping_percentage.log');
+	open(my $log, '>', $log_dir.'mapping_percentage_'.$min_depth.'x_'.$min_map.'%.log');
 	
```

Comment:
A few of the biologists wanted a way to quickly determine the filter values used for each run of the script so I added them to the file name.  Also prevents old logs with the same name but different filter criteria from being overwritten.